buildScripts/build_helper_run_unit_tests

```diff
@@ -22,7 +22,7 @@ The environment variable ANDROID_SDK_ROOT needs to be set.""")
 build_helper_functions.check_number_of_parameters(valid_param_count_min=0, valid_param_count_max=1, usage_func=usage)
 
 ## RUN tests
-test_runner_cmd = [ build_helper_functions.get_relative_gradle_name(), '-Pjenkins', 'clean', 'testDebug' ]
+test_runner_cmd = [ build_helper_functions.get_relative_gradle_name(), '-Pjenkins', 'clean', 'jacocoTestDebugUnitTestReport' ]
 ## pass filter if argument is given
 if len(sys.argv) > 1:
```

Comment:
jacocoTestDebugUnitTestReport does not seem to provide the '--tests' filter. Therefore the optional argument can be removed. As consequence 'build_step_run_unit_tests__all_tests' as a wrapper with defined arguments to 'build_helper_run_unit_tests' would be useless and they could be merged in an 'build_step_run_unit_tests'.

Reply:
Will do.